### PR TITLE
feature: remove trimap and divide operation with bimap

### PR DESCRIPTION
**Pull request by YuriCat**
Though `trimap_r` is useful in some cases, it's enough to combine multiple `bimap_r`.

### CODE REVIEW

**Comment by YuriCat** [comment]
I found that there are ternary operators such as `torch.cat(x, y, dim)`.
Therefore, not only `trimap` but also `quadmap, quinmap, ...` would be necessary in some cases.

**Comment by YuriCat** [comment]
An idea:
```
def dualmap_r([x, y, z, ...], lambda h: h[0] * h[1] * h[2] ...)
```
